autokeras/hypermodel/hyper_block.py

```diff
@@ -102,40 +96,91 @@ def build(self, hp, inputs=None):
         input_node = tf.reshape(input_node, [-1, shape[1], feature_size])
         output_node = input_node
 
-        in_layer = layer_utils.get_rnn_block(
-            hp.Choice('rnn_type', ['vanilla', 'gru', 'lstm'], default='vanilla'))
+        in_layer = const.Constant.RNN_LAYERS[hp.Choice('rnn_type',
+                                                       ['gru', 'lstm'],
+                                                       default='lstm')]
         choice_of_layers = hp.Choice('num_layers', [1, 2, 3], default=2)
 
-        for i in range(choice_of_layers):
-            return_sequences = False if i == choice_of_layers - 1 else True
-            bidirectional_block = tf.keras.layers.Bidirectional(
-                in_layer(feature_size, activation='tanh',
-                         return_sequences=return_sequences))
-            output_node = bidirectional_block(output_node)
+        bidirectional = self.bidirectional
+        if bidirectional is None:
+            bidirectional = hp.Choice('bidirectional',
+                                      [True, False],
+                                      default=True)
```

Comment:
This line is potentially making `bidirectional` and `self.bidirectional` inconsistent